lib/Workflow/Manager.php

```diff
@@ -333,4 +334,74 @@ public function getTransitionByName(string $workflowName, string $transitionName
 
         return null;
     }
+
+    /**
+     * Compatibility Fix: forces an initial place being set (and stored) if the current place is empty.
+     * We cannot apply a regular transition b/c it would be considered invalid by the state machine.
+     *
+     * As of Symfony 4.4.8 built-in implementations of @see \Symfony\Component\Workflow\MarkingStore\MarkingStoreInterface
+     * use strict `null` comparison when retrieving the current marking. This will throw exceptions for elements
+     * stored with empty marking despite an initial state was configured in prior versions of Pimcore.
+     *
+     * @param Asset|Concrete|PageSnippet $subject
+     *
+     * @return bool true if initial state was applied
+     *
+     * @throws \Exception
+     */
+    public function ensureBackwardCompatibleInitialPlace(string $workflowName, $subject): bool
```

Comment:
Actually this is not only a b/c fix, but also is needed, when a new workflow is configured for existing data objects. Or do I miss something there? 
So maybe we should name this method something like `ensureInitialPlace`

Reply:
True, if a new workflow with initial place is added _or_ an initial place is configured for a workflow that had none previously, all existing objects need to be handled. Will change the method name 👍